### PR TITLE
Hypervisor dmsg tab.

### LINKED ISSUE #407: HypervisorUI dmsg tab
**Feature description**

@skycoin-main requested that we add a `dmsg` tab to the landing page of the hypervisor UI. This tab should display:

The visors connected to the hypervisor (same list as in the Visor list tab). Below each visor, the UI should display the PubKey of the dmsg.Server used to connect to the visor as well as the Ping roundtrip time. 

Requires: https://github.com/SkycoinProject/dmsg/issues/110

### PR DESCRIPTION
**Pull request by evanlinjin**
Fixes #407

 Changes:	
- Added `/api/dmsg` endpoint to Hypervisor which tracks `dmsgctrl` connections.
- Visors now serves `dmsgctrl` streams.
- Fix segmentation fault due to incorrect address resolver init logic in `snet`.

How to test this PR:
- Start interactive testing environment with:
  - `skywire-services@feature/use-promutil`
  - `dmsg@develop`
- Navigate t0 `https://localhost:8000/api/dmsg`
- Attempts can be made to start/stop different visors and see changes to the output given by `https://localhost:8000/api/dmsg`. Note that some changes are not shown immediately.

### CODE REVIEW

File: cmd/hypervisor/commands/root.go [hunk 1]
```diff
@@ -117,7 +121,25 @@ func prepareConfig(args []string) (conf hypervisor.Config) {
...
+	return dmsgC
+}
+
+func serveDmsg(ctx context.Context, hv *hypervisor.Hypervisor, conf hypervisor.Config) {
+	go func() {
+		if err := hv.ServeRPC(ctx, conf.DmsgPort); err != nil {
+			log.WithError(err).Fatal("Failed to serve RPC client over dmsg.")
+		}
+	}()
+	log.WithField("addr", fmt.Sprintf("dmsg://%s:%d", conf.PK, conf.DmsgPort)).
```
Comment (nkryuchkov) [inline cmd/hypervisor/commands/root.go]: What do you think about this instead?
```suggestion
	log.WithField("addr", dmsg.Addr{PK: conf.PK, Port: conf.DmsgPort}).
```

File: pkg/hypervisor/dmsg_tracker.go [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+			updateAllTrackers(ctx, dtm.dm)
+			dtm.mx.Unlock()
+
+			cancel()
+		}
+	}
+}
+
+func updateAllTrackers(ctx context.Context, dts map[cipher.PubKey]*DmsgTracker) {
+	log := log.WithField("func", "DmsgTrackerManager.updateAllTrackers")
```
Comment (nkryuchkov) [inline pkg/hypervisor/dmsg_tracker.go]: What do you think of using `reflect` package to detect the function name? Hard-coding may lead to a forgotten modification if the function is renamed.

**Review by nkryuchkov — APPROVED** [review]
Good work!

File: pkg/hypervisor/dmsg_tracker.go (hunk 2 above)
Comment (evanlinjin) [reply pkg/hypervisor/dmsg_tracker.go]: @nkryuchkov I'll have a look into how it should be done

**Pull request merged**